src/test/groovy/org/prebid/server/functional/tests/privacy/GdprAmpSpec.groovy

```diff
@@ -369,4 +370,64 @@ class GdprAmpSpec extends PrivacyBaseSpec {
         assert response.ext?.warnings[PREBID]*.message ==
                 ["Parsing consent string: ${tcfConsent} failed. TCF policy version ${invalidTcfPolicyVersion} is not supported" as String]
     }
+
+    def "PBS amp should try to fetch vendor list by exponential backoff and emit error when vendor list response is absent"() {
+        given: "Test start time"
+        def startTime = Instant.now()
+
+        and: "Prepare tcf consent string"
+        def tcfConsent = new TcfConsent.Builder()
+                .setPurposesLITransparency(BASIC_ADS)
+                .setTcfPolicyVersion(tcfPolicyVersion.value)
+                .setVendorListVersion(tcfPolicyVersion.vendorListVersion)
+                .setVendorLegitimateInterest([GENERIC_VENDOR_ID])
+                .build()
+
+        and: "AMP request"
+        def ampRequest = getGdprAmpRequest(tcfConsent)
+
+        and: "Default stored request"
+        def ampStoredRequest = BidRequest.defaultBidRequest
+
+        and: "Stored request in DB"
+        def storedRequest = StoredRequest.getStoredRequest(ampRequest, ampStoredRequest)
+        storedRequestDao.save(storedRequest)
+
+        and: "Reset valid vendor list response"
+        vendorListResponse.reset()
+
+        and: "Set vendor list response with delay"
+        vendorListResponse.setResponse(tcfPolicyVersion, Delay.seconds(3))
```

Comment:
rather use time based of `exponential-backoff` variable, to make it more clear

`Delay.seconds(exponential-backoff + n)`